### PR TITLE
Add IRC link to README.md

### PR DESCRIPTION
**Pull request by ghost**
I added IRC link README.md, now it is easier to enter the IRC.

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Not sure if we should add a webchat link. It could lead to users joining the dev channel when the have end user problem...

**Comment by paveljanik** [comment]
Concept NACK

No webchat links please.

**Comment by laanwj** [comment]
On one hand it is nice to make it easy to join the chat, on the other hand one of the reasons that we moved from `bitcoin-dev` to `bitcoin-core-dev` is that the former was getting too noisy. The new IRC channel is for development-related talk only, not for questions or support or ancillary discussion about bitcoin or bitcoin core. End-users should join `#bitcoin` not `#bitcoin-core-dev`.

So tend to agree with @jonasschnelli's sentiment.

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The developer [mailing list](https://lists.linuxfoundation.org/mailman/listinfo/
 should be used to discuss complicated or controversial changes before working
 on a patch set.
 
-Developer IRC can be found on Freenode at #bitcoin-core-dev.
+Developer IRC can be found on Freenode at [#bitcoin-core-dev](https://webchat.freenode.net/?channels=#bitcoin-core-dev).
```
Comment (MarcoFalke) [inline README.md]: I am not convinced we should mention the dev chat in the readme...

We already link to CONTRIBUTING.md just three lines above. Do you mind to move the `Developer IRC can be found on Freenode at #bitcoin-core-dev. ` to CONTRIBUTING.md ?

Comment (laanwj) [inline README.md]: Sounds good to me.

**Comment by mcccs** [comment]
@paveljanik NACK rejected

> NACK means "I disagree this should be merged", and must be accompanied by sound technical justification. NACKs without accompanying reasoning may be disregarded;

You had to provide technical reasons. Mine is a document, so it is impossible to provide "technical" justification.

**Comment by laanwj** [comment]
With the move to CONTRIBUTORS I think it's okay.

**Comment by jonasschnelli** [comment]
Not sure about this change....
Moving the IRC channel away from the mailing list link and adding a webchat URL (IMO devs should not use webchats) looks after a step backward.

**Comment by MarcoFalke** [comment]
Meh, then let's just leave it as is?

**Comment by paveljanik** [comment]
I think that moving link to #bitcoin-dev-core to CONTRIBUTING.md is good. Let's keep it (without webchat link).

**Comment by MarcoFalke** [comment]
So with the same reason we could move the mailing list to CONTRIBUTING.md?

Anyway, why is the `bitcoin-core-dev` mailing list mentioned nowhere?

**Comment by laanwj** [comment]
This is seemingly too controversial for such a simple change. Also seems that the OP deleted their account. Closing.

**Pull request closed without merging**